<launch>
  <arg name="launch_prefix" default="" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="node_namespace" default="ubiquityrobot" />

  <group ns="$(arg node_namespace)">
      <node pkg="pi_driver" type="pi_driver_node.py" output="screen" name="pi_driver_node" />
  </group>

  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />

  <node pkg="web_video_server" type="web_video_server" name="web_video_server_1" output="screen">
    <param name="port" type="int" value="8080" />
    <param name="address" type="string" value="0.0.0.0" />
    <param name="server_threads" type="int" value="1" />
    <param name="ros_threads" type="string" value="2" />

    <param name="width" type="int" value="1280" />
    <param name="height" type="int" value="1080" />
    <param name="quality" type="int" value="90" />

  </node>

</launch>
